django/contrib/gis/geos/geometry.py

```diff
@@ -174,12 +174,12 @@ def from_gml(cls, gml_string):
     def __eq__(self, other):
         """
         Equivalence testing, a Geometry may be compared with another Geometry
-        or a WKT representation.
+        or a WKT or EWKT representation.
```

Comment:
I didn't know EWKT was a PostGIS specific format. Do you think it's still a good idea to compare on EWKT?